tooling-tests/src/test/java/io/ballerina/persist/tools/ToolingTestUtils.java

```diff
@@ -0,0 +1,95 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.persist.tools;
+
+import io.ballerina.persist.cmd.Init;
+import io.ballerina.projects.ProjectEnvironmentBuilder;
+import io.ballerina.projects.environment.Environment;
+import io.ballerina.projects.environment.EnvironmentBuilder;
+import org.testng.Assert;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * persist tool test Utils.
+ */
+public class ToolingTestUtils {
+
+    private static PrintStream errStream = System.err;
+
+    public static final String SAMPLES_DIRECTORY = "input";
```

Comment:
Seems this isn't used.

Reply:
Fixed in [181e367](https://github.com/ballerina-platform/persist-tools/pull/9/commits/181e36711c0bbd5cde7ab0dd0fce2b1445b72e65)